cmd/podman/root.go

```diff
@@ -580,6 +580,8 @@ func rootFlags(cmd *cobra.Command, podmanConfig *entities.PodmanConfig) {
 
 		pFlags.BoolVar(&podmanConfig.TransientStore, "transient-store", false, "Enable transient container storage")
 
+		pFlags.StringArrayVar(&podmanConfig.PullOptions, "pull-option", nil, "Tweak how an image is pulled")
```

Comment:
if this can be seen by users, it probably should be something that uses words like "custom storage options" overrides, etc.?

Reply:
it is not visible, but better to improve it so it is ready when/if decide to make it visible.

Changed in the new version